--- rocksdb-4.6.1/build_tools/build_detect_platform~	2016-05-07 00:29:53.000000000 +0000
+++ rocksdb-4.6.1/build_tools/build_detect_platform	2016-06-21 20:06:10.808749066 +0000
@@ -368,7 +368,7 @@
   elif test -n "`echo $TARGET_ARCHITECTURE | grep ^s390x`"; then
     COMMON_FLAGS="$COMMON_FLAGS -march=z10 "
   else
-    COMMON_FLAGS="$COMMON_FLAGS -march=native "
+    COMMON_FLAGS="$COMMON_FLAGS -march=westmere "
   fi
 fi
 
